### PR TITLE
Release 1.54.0 Added FA REST 2.35

### PR DESCRIPTION
**Pull request by jakubstilec**

### CODE REVIEW

File: pypureclient/flasharray/FA_2_0/client.py [hunk 1]
```diff
@@ -18,15 +18,15 @@
 
 
 class Client(object):
-    DEFAULT_TIMEOUT = 15.0
+    DEFAULT_TIMEOUT = 15
```
Comment (jakubstilec) [inline pypureclient/flasharray/FA_2_0/client.py:21]: missing removal of timeout

File: docs/requirements.txt [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 docutils<0.18
 six >=1.10, <=1.16.0
 python_dateutil >=2.5.3
-setuptools >=21.0.0, <=68.1.2
+setuptools >=70.0.0
 urllib3 >= 1.26.17
-paramiko >=2.11.0, <= 2.12.0
+paramiko >= 3.4.0
 pyjwt >=2.0.0, <=2.8.0
 requests >=2.20.1, <=2.31.0
```
Comment (jakubtischler) [inline docs/requirements.txt:9]: Let's also update this version.

Otherwise it looks ok.

File: pypureclient/flasharray/FA_2_0/client.py (hunk 1 above)
Comment (jakubstilec) [reply pypureclient/flasharray/FA_2_0/client.py]: fixed

File: docs/requirements.txt (hunk 2 above)
Comment (jakubstilec) [reply docs/requirements.txt]: updated

**Review by jakubtischler — APPROVED** [review]

**Comment by jakubtischler** [comment]
@mhruska 
The files renamed are currently unused. This is being dealt with on a larger scale - I would suggest to leave the changes for now and fix this in the next release.

File: pypureclient/flasharray/FA_2_10/models/smtp.py [hunk 3]
```diff
@@ -21,7 +21,7 @@
 if typing.TYPE_CHECKING:
     from pypureclient.flasharray.FA_2_10 import models
 
-class SMTP(object):
+class Smtp(object):
```
Comment (Gregory-Bessonov) [inline pypureclient/flasharray/FA_2_10/models/smtp.py:24]: FYI: These classes are available to end-users, so renaming them might cause some issues.

File: pypureclient/flasharray/FA_2_13/client.py [hunk 4]
```diff
@@ -18,15 +18,14 @@
 
 
 class Client(object):
-    DEFAULT_TIMEOUT = 15.0
```
Comment (Gregory-Bessonov) [inline pypureclient/flasharray/FA_2_13/client.py:21]: nit: It's worth mentioning in release notes as a bug fix

Comment (jakubtischler) [reply pypureclient/flasharray/FA_2_13/client.py:21]: Will do, good point.

**Pull request merged**